src/pages/Admin/Admin.tsx

```diff
@@ -398,26 +373,29 @@ const VenueInfoComponent: React.FC<VenueInfoComponentProps> = ({
             </button>
             <Link
               to={{ search: "manageUsers=true" }}
-              className="btn btn-primary"
+              className="btn btn-block btn-primary"
             >
               Manage Venue Owners
             </Link>
             {typeof roomIndex !== "number" && (
-              <div>
+              <div className="page-container-adminpanel-actions-note">
                 If you are looking to edit one of your rooms, please select the
                 room in the left hand menu
               </div>
             )}
+            {canBeDeleted(venue) && (
+              <button
+                role="link"
+                className="btn btn-block btn-danger"
+                onClick={show}
```

Comment:
```suggestion
                onClick={showDeleteModal}
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1380/commits/7105aeff532bea24d5bb7b8ca47f2fb586e955d7